images/busybox/main.tf

```diff
@@ -0,0 +1,77 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+# Create another apko provider specifically for alpine, without the global wolfi defaults
+provider "apko" {
+  alias = "alpine"
+
+  default_archs = [] # this defaults to "all"
+  extra_keyring = ["https://dl-cdn.alpinelinux.org/alpine/edge/main"]
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  providers = {
+    apko = apko.alpine
+  }
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.apko.yaml")
+  extra_packages    = []
+}
+
+module "latest-glibc" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.glibc.apko.yaml")
+}
+
+module "version-tags" {
+  source  = "../../tflib/version-tags"
+  package = "busybox"
+  config  = module.latest.config
+}
+
+module "version-tags-glibc" {
+  source  = "../../tflib/version-tags"
+  package = "busybox"
+  config  = module.latest-glibc.config
+}
+
+module "test-latest" {
+  source = "./tests"
+  digest = module.latest.image_ref
+}
+
+module "test-latest-glibc" {
+  source = "./tests"
+  digest = module.latest-glibc.image_ref
+}
+
+module "tagger" {
```

Comment:
Could we move these into latest.tf and latest-glibc.tf? It's a bit tangled all together like this.